spread.yaml

```diff
@@ -84,6 +84,7 @@ exclude:
 
 prepare-each: |
     journalctl --rotate
+    sleep .1
```

Comment:
Let's reduce it further on a follow up. Otherwise for 250 tests we lose 25 seconds just here.